README.md

```diff
@@ -1,2 +1,431 @@
+  import Page from 'components/layouts/Page';
+  ```
+
+## JSX
+
+JSX leverages syntactic similarity with HTML so similar conventions should be observed.
+
+* rendering
+  ```js
+  // good
+  render() {
+    return <Button text="Save" onChange={this.save}/>;
+  }
+  // good
+  render() {
+    return (
+      <div className="action-bar">
+        <Button text="Save" onChange={this.save}/>
+        <Button text="Cancel" onChange={this.goBack}/>
+      </div>
+    );
+  }
+
+  // bad
+  render() {
+    return <div className="action-bar">
+      <Button text="Save" onChange={this.save}/>
+      <Button text="Cancel" onChange={this.goBack}/>
+    </div>;
+  }
+  ```
+
+* use double quotes for attributes
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button text='Save' onChange={this.save}/>
+  ```
+
+* Do not use braces around double quotes
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button text={"Save"} onChange={this.save}/>
+  ```
+
+* Do not use spaces around =
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button text = "Save" onChange = {this.save}/>
+  ```
+
+* Inline short components
+  ```js
+  // good
+  <Button text="Save" onChange={this.save}/>
+
+  // bad
+  <Button
+    text="Save"
+    onChange={this.save}/>
+  ```
+
+* Multiline each prop on it's own line
+  ```js
+  // good
+  <Input
+    type="email"
+    label="Email"
+    placeholder="name@example.com"
+    onChange={this.setValue('email')}
+    onClick={this.selectText}/>
+
+  // bad
+  <Input
+    type="email" label="Email" placeholder="name@example.com"
+    onChange={this.setValue('email')} onClick={this.selectText}/>
+  ```
+
+* Keys should come first and may be on the first line when multilined.
+  ```js
+  // good
+  <ul>
+    {cars.map((car) => (
+      <Car key={car.id}
+        car={car}
+        detail="list"
+        showPrice={true}/>
+    ))}
+  </ul>
+
+  // bad
+  <ul>
+    {cars.map((car) => (
+      <Car
+        car={car}
+        detail="list"
+        key={car.id}
+        showPrice={true}/>
+    ))}
+  </ul>
+  ```
+
+* Multiline contents if there are nested elements.
+  ```js
+  // good
+  <p>Hi {name}, welcome to the site!</p>
+  // good
+  <p>
+    <i className="icon-checkmark"/>
+    <strong>Hi {name}</strong>, welcome to the site!
+  </p>
+
+  // bad
+  <p><i className="icon-checkmark"/><strong>Hi {name}</strong>, welcome to the site!</p>
+  ```
+
+* Extract components, don't have multiple rendering functions. This will make the class shorter and more readable and make it easier to extract and reuse common components.
+  ```js
+  // good
+  const ErrorMessage = (message) => {
+    if (!message) return null;
+    return <div className="error"><i className="icon-alert"/>{message}</div>;
+  }
+
+  class MyComponent extends React.Component {
+    render() {
+      const {cars, error} = this.props;
+      return (
```

Comment:
What do think about a space between the things defined in the render before the return & the return statement?

Reply:
I frequently do. I'm not sure I want to make it a recommendation